Sources/DDGSync/SyncFeatureFlags.swift

```diff
@@ -24,6 +24,7 @@ import Foundation
  */
 public struct SyncFeatureFlags: OptionSet {
     public let rawValue: Int
+    public var unavailableReason: PrivacyConfigurationFeatureDisabledReason?
```

Comment:
This should be a `public private(set) var`, so that it can't be modified from outside.

Reply:
Yes! def should have done that